modules/build/src/test/scala/scala/build/tests/InputsTests.scala

```diff
@@ -1,8 +1,10 @@
 package scala.build.tests
 
 import com.eed3si9n.expecty.Expecty.expect
+import os.Path
```

Comment:
Let's not import that one, so that we don't confuse it with NIO paths.

Reply:
Should be fixed